account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = 'account.move.line'
+
+    is_tax_editable = fields.Boolean(
+        string="Is tax data editable?", compute='_compute_is_tax_editable')
+
+    @api.multi
+    @api.depends('move_id.state')
```

Comment:
Depends is **only** triggered for stored fields...do we really want to have it stored?
If we want to and it somehow makes sense it would be much easier to use a related field for posted and use this instead and make the field readonly in the ```account.move.line``` - this would also remove all the overhead added in your logic.